ci/openshiftio-appdev-docs/src/main/java/io/openshift/appdev/documentation/builder/HttpApplication.java

```diff
@@ -22,26 +22,40 @@
 import io.vertx.ext.web.handler.RedirectAuthHandler;
 import io.vertx.ext.web.handler.StaticHandler;
 
+import java.io.IOException;
+import java.util.Properties;
+
 public class HttpApplication extends AbstractVerticle {
 
-   private static final String INDEX_PAGE = "index.html";
-   private static final String LAUNCHER_TEMPLATE_LATEST_URL = "https://raw.githubusercontent.com/openshiftio/launchpad-templates/v5/openshift/launchpad-template.yaml";
+    private static final String propFileLocation = "/application.properties";
+    private static final Properties props = loadProperties();
+
+    private static Properties loadProperties() {
+        Properties props = new Properties();
+        try {
+            props.load(HttpApplication.class.getResourceAsStream(propFileLocation));
```

Comment:
You could also lookup both configurables from props and store them into static variables here, but I don't think that's a big deal. Just an idea.